terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/ExistInCodeListTest.java

```diff
@@ -398,9 +433,8 @@ public String getGenderLabel(String genderCode) {
 }
 
 class Order {
-    @ExistInCodeList.List(value = {
-            @ExistInCodeList(codeListId = "CD_MULTIPLES_OF_3", message = "number must be multiples of 3"),
-            @ExistInCodeList(codeListId = "CD_EVEN", message = "number must be even") })
+    @ExistInCodeList(codeListId = "CD_MULTIPLES_OF_3", message = "number must be multiples of 3")
+    @ExistInCodeList(codeListId = "CD_EVEN", message = "number must be even")
```

Comment:
Need to reflect Guideline. ( not need change, but changeable )

Reply:
I agree .